ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -569,19 +569,6 @@ int window_get_cursor()
   return enigma::cursorInt;
 }
 
-void io_handle()
-{
-  MSG msg;
-  enigma::input_push();
-  while (PeekMessage (&msg, NULL, 0, 0, PM_REMOVE))
-  {
-    TranslateMessage (&msg);
-    DispatchMessage (&msg);
-  }
```

Comment:
I don't think this ever worked because it's not actually running our window procedure.